### ISSUE TITLE
visual genome test, mAP close to 0 ????

### ISSUE THREAD

**Issue by akira-l** [jwyang/faster-rcnn.pytorch#283]
training vg dataset using this repo in 4*P40. I got the mAP close to 0 (0.0006/0.0004/0)

using the given model (vgg16 in vg), which should have 4.4 mAP, but I got 0.4 (0.0004) after test the model by 'test_net.py' 

I try the pascal voc dataset as well, and got 48.84 in mAP, which ought to be 69 at least.

**Comment by akira-l** [comment]
anyone have met the same problem?

**Comment by akira-l** [comment]
using python3 pytorch0.4.0 I fix some bugs which already proposed and sovled in other issue. 
I'm not sure what or where the problem is. training procedure? test function? data preprocessing? anyone met the same problem? or even tell me how to find the problem? thx!!!!

**Comment by akira-l** [comment]
using python3 pytorch0.4.0 I fix some bugs which already proposed and sovled in other issue.
I'm not sure what or where the problem is. training procedure? test function? data preprocessing? anyone met the same problem? or just give me some advice or tell me how to find the problem? thx!!!!

**Comment by akira-l** [comment]
@jwyang

**Comment by jwyang** [comment]
@akira-l I will re-evaluate the pertrained model. Have you ever run the training code?

**Comment by akira-l** [comment]
yes, training in pascal voc and visaul genome. Got 48.84 mAP in VOC and 0 mAP in VG.

**Comment by akira-l** [comment]
any advice? @jwyang 
thx

**Comment by jwyang** [comment]
it is weird in this case, I trained the model yesterday, and the results match the reported number in the benchmarking table. I just also tested the [pretrained model](
https://www.dropbox.com/s/6ief4w7qzka6083/faster_rcnn_1_6_10021.pth?dl=0), the numbers match to the table.

**Comment by akira-l** [comment]
thx! @jwyang  What's the environment in detail? I can not run the programm directly after clone this repo. So, I fix some error like 'ZeroDivisionError', etc. , but I don't think these changing in code would lead to such weird condition. 
So, if you can run the code without changing anything, may be the python or pytorch version or other enviroment have something wrong. 
Otherwise,  there are some problems in my data. Maybe the xml files are not correct.

**Comment by jwyang** [comment]
@akira-l mine is Titan Xp, cuda-8.0, pytorch-0.4.0, python-2.7.

**Comment by zc-alexfan** [comment]
@akira-l Did you manage to resolve it?

**Comment by alibabadoufu** [comment]
I have the same problem. Reported mAP is 0.4% only
